### PR TITLE
[TIC-136] Updated protected route to require admin permissions

### LINKED ISSUE #446: Can Access Admin Pages via direct URLs as non-admin
When logged-in to a non-admin account, although the options to go to the admin pages do not appear on the customer-facing end, users are able to directly type in the URL to admin pages and then edit things as they wish.

APIs will also work for non-admin despite checks for `admin`.

### PR DESCRIPTION
**Pull request by benfaustin95**
### Description
Updated the ProtectedRoute component to verify that a user is an admin in addition to the current
verification that they are logged in.

### Risks
NA

### Validation
Manual testing

### Issue
Closes #446 

### Operating System
M1 Mac

### CODE REVIEW

File: client/src/components/Authentication/protected-route.tsx [hunk 1]
```diff
@@ -1,25 +1,32 @@
 /*
 This component takes a component in as a prop,
-and only retrns it if one is logged in.
+and only returns it if one is logged in and an admin.
 */
```
Comment (henrykaus) [inline client/src/components/Authentication/protected-route.tsx:4]: Could you move this comment to replace the one associated with the `ProtectedRoute` component below on line 9-10

**Review by henrykaus — COMMENTED** [review]
This works great! A good way to disallow access to the admin pages for non-admin. I'm going to hold off on approval until the Auth0 situation is resolved so we can keep testing in the deployed environments. I also noticed that there is a duplicate `/` Route in `App.tsx`; do you think you could remove that as part of this PR?

**Comment by benfaustin95** [comment]
> This works great! A good way to disallow access to the admin pages for non-admin. I'm going to hold off on approval until the Auth0 situation is resolved so we can keep testing in the deployed environments. I also noticed that there is a duplicate `/` Route in `App.tsx`; do you think you could remove that as part of this PR?

Sounds good

**Review by henrykaus — APPROVED** [review]

**Comment by henrykaus** [comment]
> > This works great! A good way to disallow access to the admin pages for non-admin. I'm going to hold off on approval until the Auth0 situation is resolved so we can keep testing in the deployed environments. I also noticed that there is a duplicate `/` Route in `App.tsx`; do you think you could remove that as part of this PR?
> 
> Sounds good

Auth0 issues have been resolved, feel free to merge

**Pull request merged**